src/Root.tsx

```diff
@@ -39,23 +37,39 @@ const globalStyles = (
   />
 );
 
+const ThemeWrapper = () => {
+  const { data: currentAccount } = hooks.useCurrentMember();
+  return (
+    <ThemeProvider
+      langs={langs}
+      languageSelectSx={{ mb: 2, mr: 2 }}
+      i18n={i18nConfig}
+      defaultDirection={i18nConfig.dir(
+        currentAccount?.type === AccountType.Individual
+          ? (currentAccount.extra.lang ?? DEFAULT_LANG)
+          : DEFAULT_LANG,
+      )}
+    >
+      <CssBaseline />
+      <Router>
+        <ErrorBoundary fallback={<FallbackComponent />}>
+          <CurrentMemberContextProvider>
+            <App />
+          </CurrentMemberContextProvider>
+        </ErrorBoundary>
+      </Router>
+    </ThemeProvider>
+  );
+};
```

Comment:
I already put a comment about this when it was implemented by Lina, but I do not really like that is is an opaque/black-box component. I would prefer for it to take a children prop and simply wrap it with the required logic.

Reply:
I revert then, it's because `theme` was deprecated.